package.json

```diff
@@ -11,7 +11,24 @@
   },
   "homepage": "https://github.com/Financial-Times/o-email-only-signup",
   "devDependencies": {
-    "origami-build-tools": "^4.2.0"
+ 	  "babel-loader": "^5.3.2",
```

Comment:
package.json uses spaces for indentation, could we change this tab to some spaces